flight/PiOS/Common/pios_mpu.c

```diff
@@ -216,8 +217,6 @@ static int32_t PIOS_MPU_Validate(struct pios_mpu_dev *dev)
 		return -1;
 	if (dev->magic != PIOS_MPU_DEV_MAGIC)
 		return -2;
-	if (dev->com_driver_id == 0)
-		return -3;
 	return 0;
```

Comment:
I dislike code that checks for things that should never happen (something being null that is not allowed to be null / would have accessed invalid memory during initialization if it was null, etc) and tries to salvage the situation when we're already deeply into undefined behavior.